src/Symfony/Component/AccessToken/AccessToken.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AccessToken;
+
+/**
+ * @author Pierre Rineau <pierre.rineau@processus.org>
+ */
+class AccessToken implements AccessTokenInterface
+{
+    /**
+     * Identifier for tokens that are not generated using credentials.
+     */
+    public const IN_MEMORY = 'in_memory';
+
+    protected ?\DateTimeImmutable $expiresAt;
+    protected ?bool $hasExpired = null;
+
+    /**
+     * @param string $id Identifier of credentials used for generating it
+     */
+    public function __construct(
+        protected readonly string $value,
+        protected readonly string $type = 'Bearer',
+        protected readonly int $expiresIn = 600,
+        protected readonly \DateTimeImmutable $issuedAt = new \DateTimeImmutable(),
+        protected readonly string $id = self::IN_MEMORY,
+    ) {
+    }
+
+    #[\Override]
```

Comment:
AFAIK we don't use this attribute in our codebase

Reply:
And maybe that's the right moment to change ?

Reply:
You can open an issue about this topic, because its out of scope for this PR IMHO

Reply:
OK thanks, I'm unsure about this convention, what should I do, an @inheritdoc PHP doc or just nothing ?

Reply:
It seems it's simply nothing. I'll simply remove those attributes.